docs/pre_release_features.md

```diff
@@ -46,30 +46,30 @@ If you'd like to review the changes for a nightly release, you can use the [comp
+- **beta\_**: this is where all new features land before they becoming part of Streamlit core. This gives you a chance to try the next big thing we're cooking up weeks or months before we're ready to stabilize its API.
+- **experimental\_**: this is where we'll put features that may or may not ever make it into Streamlit core. We don't know whether these features have a future, but we want you to have access to everything we're trying, and work with us to figure them out.
 
-The main difference between `st.beta` and `st.experimental` is that beta features are expected to make it into the `st` namespace at some point soon, while experimental features may never make it.
+The main difference between `beta_` and `experimental_` is that beta features are expected to make it into Streamlit core at some point soon, while experimental features may never make it.
 
 ### Beta
 
-Features in the beta namespace are all scheduled to become part of `st`, or core Streamlit. While in beta, a feature's API and behaviors may not be stable, and it's possible they could change in ways that aren't backward-compatible.
+Features with the `beta_` naming convention are all scheduled to become part of Streamlit core. While in beta, a feature's API and behaviors may not be stable, and it's possible they could change in ways that aren't backward-compatible.
 
 **The lifecycle of a beta feature**
 
-1. A feature is added to the beta namespace.
-2. The feature's API stabilizes and the feature is _cloned_ into the `st` namespace, so it exists in both st and `st.beta`. At this point, users will see a warning when using the version of the feature that lives in the beta namespace -- but the `st.beta` feature will still work.
-3. At some point, the feature is _removed_ from the `st.beta` namespace, but there will still be a stub in `st.beta` that shows an error with appropriate instructions.
-4. Finally, at a later date the stub in `st.beta` is removed.
+1. A feature is added with the `beta_` prefix.
+2. The feature's API stabilizes and the feature is _cloned_ without the `beta_` prefix, so it exists as both `st` and `beta_`. At this point, users will see a warning when using the version of the feature with the `beta_` prefix -- but the feature will still work.
+3. At some point, the code of the `beta_`-prefixed feature is _removed_, but there will still be a stub of the function prefixed with `beta_` that shows an error with appropriate instructions.
+4. Finally, at a later date the `beta_` version is removed.
 
 ### Experimental
 
-Features in the experimental namespace are things that we're still working on or trying to understand. If these features are successful, at some point they'll become part of core Streamlit, by moving to the `st.beta` namespace and then to `st`. If unsuccessful, these features are removed without much notice.
```

Comment:
"with the `experimental_` naming convention"?

Reply:
:+1: